modules/apiserversource-kn-delete.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// serverless/knative_eventing/serverless-kn-source.adoc
+
+[id="delete-apiserversource-kn_{context}"]
+= Deleting the ApiServerSource using the Knative CLI (`kn`)
+
+You can delete the ApiServerSource, trigger, service, service account, cluster role, and cluster binding created in this guide by entering the following `kn` and `oc` commands.
```

Comment:
I'd reword this slightly:

> You can delete the ApiServerSource, trigger, service, service account, cluster role, and cluster binding using `kn` and `oc` commands.

Or better yet, match the wording used in the next section:

> This section describes the steps used to delete the ApiServerSource, trigger, service, service account, cluster role, and cluster binding using `kn` and `oc` commands.